tests/ttnn/unit_tests/operations/test_binary_composite.py

```diff
@@ -534,3 +534,167 @@ def test_binary_polyval_ttnn(input_shapes, coeffs, device):
 
     comp_pass = compare_pcc([output_tensor], [golden_tensor])
     assert comp_pass
+
+
+@pytest.mark.parametrize(
+    "input_shapes",
+    (
+        (torch.Size([1, 1, 32, 32])),
+        (torch.Size([1, 1, 320, 384])),
+        (torch.Size([1, 3, 320, 384])),
+    ),
+)
+def test_binary_gti_ttnn(input_shapes, device):
+    in_data1, input_tensor1 = data_gen_with_range(input_shapes, -100, 100, device)
+    in_data2, input_tensor2 = data_gen_with_range(input_shapes, -150, 150, device)
+    ttnn.gt_(input_tensor1, input_tensor2)
+    golden_function = ttnn.get_golden_function(ttnn.gt_)
+    golden_tensor = golden_function(in_data1, in_data2)
+
+    comp_pass = compare_pcc([input_tensor1], [golden_tensor])
```

Comment:
@mouliraj-mcw for these ops we need to check the equality. Can you check the comp_pcc and updates

Reply:
Sure akka will update it with equality check.